src/Poseidon/CLI/Init.hs

```diff
@@ -47,7 +50,7 @@ runInit (InitOptions (GenotypeDataSpec format_ genoFile_ _ snpFile_ _ indFile_ _
     -- create new package
     logInfo "Creating new package entity"
     outName <- liftIO $ determinePackageOutName maybeOutName outPath
-    inds <- liftIO $ loadIndividuals outPath genotypeData
+    inds <- liftIO $ loadIndividuals outPath genotypeData plinkPopMode
```

Comment:
General observation: Maybe we will need a general data type for these genotype data settings like `plinkPopMode`. I can for example imagine that more settings may emerge if we ever come around to allow .vcf files in Poseidon packages.

Reply:
Yes, well I was hoping we could creep that into `GenotypeDataSpec`, but unfortunately that type maps 1 to 1 to the YAML, so that won't work.

Reply:
Ah - I see. I think for now it's fine to keep it as a special option. We can generalize it later when the need arises.

Reply:
Agreed.